draft-ietf-quic-recovery.md

```diff
@@ -690,9 +690,22 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
     if (sent_packets[ack.largest_acked]):
       latest_rtt = now - sent_packets[ack.largest_acked].time
       UpdateRtt(latest_rtt, ack.ack_delay)
-    // Find all newly acked packets.
+    // Find all newly acked packets and track the smallest
+    smallest_newly_acked = 2^62
     for acked_packet in DetermineNewlyAckedPackets():
+      smallest_newly_acked =
+        min(smallest_newly_acked, acked_packet.packet_number)
       OnPacketAcked(acked_packet.packet_number)
+    
+    if smallest_newly_acked != 2^62
```

Comment:
I still think we should make 0 the sentinel value.

Reply:
I was going to, but 0 is a valid packet number.

Reply:
My point is that it shouldn't be, in order to avoid ugly code like this here, and at many other places.

Reply:
I'm not disagreeing, but I don't think this is the PR to change that?